### PR TITLE
add tools.collect_libs()

### PR DESCRIPTION
**Pull request by ovidiub13**

### CODE REVIEW

File: reference/tools.rst [hunk 1]
```diff
@@ -513,3 +513,20 @@ Replaces the ``prefix`` variable in a package config file ``.pc`` with the speci
...
+
+- **conanfile**: A `ConanFile` object from which to get the `package_folder`
+- **folder**: The subfolder where the library files are. Defaulted to "lib"
+
+**Example:**
+
+.. code-block:: python
+
+     def package_info(self):
+        self.cpp_info.libs = tools.collect_libs()
```
Comment (lasote) [inline reference/tools.rst]: The example should use the "self"

Comment (ovidiub13) [reply reference/tools.rst]: but this was moved to the tools module since 0.27.

Comment (ovidiub13) [reply reference/tools.rst]: https://github.com/conan-io/conan/blob/develop/conans/client/tools/files.py#L207

Comment (ovidiub13) [reply reference/tools.rst]: http://conanio.readthedocs.io/en/latest/changelog.html?highlight=collect_libs#september-2017

Comment (ovidiub13) [reply reference/tools.rst]: Sorry, I misunderstood. I've added it now.

**Pull request merged**